collectors/ApicLeafCapacityCollector.py

```diff
@@ -37,45 +41,86 @@ def get_metrics(self, host: str, data: Dict) -> List[GaugeMetricFamily]:
                 leaf_id = leaf_dn[2]
                 if not self.leaf_ids.get(leaf_id, False):
                     continue
-                for data_object in leaf_data:
-                    if 'eqptcapacityL3TotalUsageCap5min' in data_object:
-                        l3_max = data_object['eqptcapacityL3TotalUsageCap5min']['attributes']['v4TotalEpCapMax']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'max', 'l3'],
-                                                   value=l3_max)
-                    elif 'eqptcapacityL3TotalUsage5min' in data_object:
-                        l3_total = data_object['eqptcapacityL3TotalUsage5min']['attributes']['v4TotalEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'total', 'l3'],
-                                                   value=l3_total)
-                    elif 'eqptcapacityL3Usage5min' in data_object:
-                        l3_local = data_object['eqptcapacityL3Usage5min']['attributes']['v4LocalEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'local', 'l3'],
-                                                   value=l3_local)
-                    elif 'eqptcapacityL3RemoteUsage5min' in data_object:
-                        l3_remote = data_object['eqptcapacityL3RemoteUsage5min']['attributes']['v4RemoteEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'remote', 'l3'],
-                                                   value=l3_remote)
-                    elif 'eqptcapacityL2TotalUsage5min' in data_object:
-                        l2_max = data_object['eqptcapacityL2TotalUsage5min']['attributes']['totalEpCapMax']
-                        l2_total = data_object['eqptcapacityL2TotalUsage5min']['attributes']['totalEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'max', 'l2'],
-                                                   value=l2_max)
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'total', 'l2'],
-                                                   value=l2_total)
-                    elif 'eqptcapacityL2Usage5min' in data_object:
-                        l2_local = data_object['eqptcapacityL2Usage5min']['attributes']['localEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'local', 'l2'],
-                                                   value=l2_local)
-                    elif 'eqptcapacityL2RemoteUsage5min' in data_object:
-                        l2_remote = data_object['eqptcapacityL2RemoteUsage5min']['attributes']['remoteEpLast']
-                        g_leaf_cap_tcam.add_metric(labels=[leaf_id, 'remote', 'l2'],
-                                                   value=l2_remote)
+                if self.gen1_leaves.get(leaf_id, False):
```

Comment:
I have ran this locally against our QA region and noticed the metrics for `max` and `total` were always `0`.
From the values in the debugger it looks like to me that this conditional must be reversed.
```suggestion
                if not self.gen1_leaves.get(leaf_id, False):
```